### PR TITLE
Scrumplex moments

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -149,6 +133,7 @@ async function parseMsg(e: Message) {
   );
 
   if (!cmd) {
+    // TODO: Do not read tags.json everytime there is a new message
```
Comment (d-513) [inline src/index.ts:136]: @ryanccn this literally was correct and you changed it in your „minor refactor”

File: src/utils/remoteVersions.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+// TODO: caching
+export async function getLatestMinecraftVersion(): Promise<string> {
+  const f = await fetch('https://meta.polymc.org/v1/net.minecraft/package.json');
+  const package = await f.json() as MetaPackage;
+  return package.recommended[0];
+}
+
+// TODO: caching
+export async function getLatestPolyMCVersion(): Promise<string> {
```
Comment (d-513) [inline src/utils/remoteVersions.ts:20]: Ok why remove caching it was there

Comment (Scrumplex) [reply src/utils/remoteVersions.ts:20]: because it sucked, lol.

There is probably a good caching library out there, but I dunno which one we could use.

Comment (d-513) [reply src/utils/remoteVersions.ts:20]: Idk why we need a lib for 2 request but I guess I’ll try to find one then

**Comment by DioEgizio** [comment]
Roly Poly just got scrumped

**Pull request merged**